middleware/src/handlers/handlers.go

```diff
@@ -0,0 +1,87 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	//"encoding/json"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+	//"github.com/tidwall/gjson"
+	"github.com/digitalbitbox/bitbox-base/middleware/src"
+	"github.com/digitalbitbox/bitbox-base/middleware/util/logging"
+	"github.com/sirupsen/logrus"
+)
+
+type Middleware interface {
+	Start() <-chan *middleware.MiddlewareInfoStruct
+}
+
+// Handlers provides a web api
+type Handlers struct {
+	Router     *mux.Router
+	clients    map[*websocket.Conn]bool
+	upgrader   websocket.Upgrader
+	log        *logrus.Entry
+	middleware Middleware
+	//TODO: In future this event should have a generic interface (thus only containing raw json)
```

Comment:
(Optional) We could add the person who can provide more context for the `TODO` (not necessarily the person who will do the work), i.e `TODO(TheCharlatan)` in this case.